packages/core/src/checkout/checkout-service.ts

```diff
@@ -1399,6 +1404,37 @@ export default class CheckoutService {
         return this._dispatch(action, { queueId: 'extensions' });
     }
 
+    /**
+     * Registers command handlers. Works on both host and extension sides.
+     *
+     * @alpha
+     * @param handlers - a set of handlers for checkout extension commands
+     * @returns A promise that resolves to the current state.
+     */
+    handleExtensionCommand(handlers: ExtensionCommandHandlers): Promise<CheckoutSelectors> {
+        const action = this._extensionActionCreator.handleExtensionCommand(handlers);
+
+        return this._dispatch(action);
+    }
+
+    /**
+     * Posts a message to checkout or an extension.
+     *
+     * @alpha
+     * @param event - A message event that strictly adheres to the ExtensionPostEvent type definition.
+     * @param extensionId - The ID of the extension to which the message will be sent. If no extension ID is provided,
+     * the message will be sent to the host.
+     * @returns A promise that resolves to the current state.
+     */
+    postExtensionMessage(
+        event: ExtensionPostEvent,
+        extensionId?: string,
+    ): Promise<CheckoutSelectors> {
+        const action = this._extensionActionCreator.postExtensionMessage(event, extensionId);
```

Comment:
Is this the "broadcast" method that you mentioned - to notify extension state changes? If that's the case, I'm not sure if there's a need to have a method that UCO needs to call. Could we just set up the change-detection subscriber internally and broadcast to all extensions interested in the change?

Reply:
Removed it for now.
Will circle back to `postMessage` once we get to the "broadcast" ticket.